client/tx/legacy.go

```diff
@@ -48,7 +49,11 @@ func CopyTx(tx signing.Tx, builder client.TxBuilder) error {
 
 	err = builder.SetSignatures(sigs...)
 	if err != nil {
-		return err
+		if ignoreSignatureError {
+			_ = builder.SetSignatures()
```

Comment:
Why we are calling this second time?

Reply:
In case there is any side effect of the initial `SetSignatures`. For instance maybe there were 2 signatures and 1 was successful but the other wasn't. This call clears all signatures.

Reply:
It's already merged, but  I still think we should add a comment.

Reply:
I did